taskcluster/test-cpp_16k-win-cuda-opt.yml

```diff
@@ -0,0 +1,11 @@
+build:
+  template_file: test-win-cuda-opt-base.tyml
+  dependencies:
+    - "win-amd64-gpu-opt"
+    - "test-training_16k-linux-amd64-py36m-opt"
+  test_model_task: "test-training_16k-linux-amd64-py36m-opt"
+  args:
+    tests_cmdline: "$TASKCLUSTER_TASK_DIR/DeepSpeech/ds/taskcluster/tc-cppwin-ds-tests.sh 16k --cuda"
```

Comment:
Weird to use `--` for a positional argument, I'd prefer just `cuda` to avoid a trap.

Reply:
Done